ci/pingcap_chaos_mesh_build_kind.groovy

```diff
@@ -0,0 +1,252 @@
+//
+// E2E Jenkins file.
+//
+
+import groovy.transform.Field
+
+@Field
+def podYAML = '''
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    # we pretend as tidb-operator in order not to meet tidb-operator-e2e job in the same node
+    app: tidb-operator-e2e
```

Comment:
we would share same cluster with tidb-operator-e2e, so we pretend ourselves like tidb-operator-e2e in order not to run both kind of e2e job in one node.